### ISSUE TITLE
WCAG 4.1.2 test 16 does not visualise in the PDF viewer

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#74]
[This PDF document](https://github.com/veraPDF/veraPDF-wcag-algs/files/6969699/test-document-10.pdf) generates error WCAG 4.1.2 test 16 "Paragraph structure element empty". You can click on the occurrences of this error but they don't visualise in the PDF document.

**Comment by jzuidweg** [comment]
Why was this issue marked as invalid? I just tried it again and I can still reproduce the issue with the attached PDF document. For your information, I am running the Dual Lab version of the checker on Edge Chromium. All the other errors visualise well, but the 8 instances of error WCAG 4.1.2 test 16 "Paragraph structure element empty" do not.

**Comment by bdoubrov** [comment]
Sorry, somehow my comment got lost. The issue is that the paragraph structure element here doesn't have any content at all. Not even empty strings or space only spans. It simply has zero children in the structure tree. Therefore there is no data to highlight on the page(s)

**Comment by jzuidweg** [comment]
@bdoubrov, perhaps in the case that there is nothing to display, the backend should not provide a clickable link to the occurrence. Is this difficult to implement?

**Comment by bdoubrov** [comment]
This was resolved on the error preview frontend.
